src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -305,140 +308,143 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
-        internal RecommendationResultSet GenerateBaselineRecommendations(SqlInstanceRequirements req, GetSkuRecommendationsParams parameters) {
-                RecommendationResultSet resultSet = new RecommendationResultSet();
-
-                SkuRecommendationServiceProvider provider = new SkuRecommendationServiceProvider(new AzureSqlSkuBillingServiceProvider());
-                AzurePreferences prefs = new AzurePreferences() {
-                    EligibleSkuCategories = null,       // eligible SKU list will be adjusted with each recommendation type
-                    ScalingFactor = parameters.ScalingFactor / 100.0,
-                    TargetEnvironment = TargetEnvironmentType.Production
-                };
-    
-                // generate SQL DB recommendations, if applicable
-                if (parameters.TargetPlatforms.Contains("AzureSqlDatabase"))
-                {
-                    Stopwatch sqlDbStopwatch = new Stopwatch();
-                    sqlDbStopwatch.Start();
-
-                    prefs.EligibleSkuCategories = GetEligibleSkuCategories("AzureSqlDatabase", parameters.IncludePreviewSkus);
-                    resultSet.sqlDbResults = provider.GetSkuRecommendation(prefs, req);
+        internal RecommendationResultSet GenerateBaselineRecommendations(SqlInstanceRequirements req, GetSkuRecommendationsParams parameters)
+        {
```

Comment:
I guess git messed up the diff -- presumably there's no actual change to any of this SKU recommendation code besides just reformatting the brackets here?

Reply:
Yep, git is showing too much. The only change was the opening curly brace moved to the next line.